assets/js/googlesitekit/datastore/user/utils.js

```diff
@@ -0,0 +1,104 @@
+/**
```

Comment:
Instead of creating a single `utils.js` file, I think it would be a bit cleaner to create a `util` subdirectory and have one file per function. The tests could also then be split across files in the same way.